### ISSUE TITLE
Modal window backround is rendered inside the modal window in front of content

### ISSUE THREAD

**Issue by Grantim** [ocornut/imgui#5532]
```
Dear ImGui 1.89 WIP (18805)
--------------------------------
sizeof(size_t): 8, sizeof(ImDrawIdx): 4, sizeof(ImDrawVert): 20
define: __cplusplus=199711
define: IMGUI_DISABLE_OBSOLETE_FUNCTIONS
define: _WIN32
define: _WIN64
define: _MSC_VER=1929
define: _MSVC_LANG=202004
--------------------------------
io.BackendPlatformName: imgui_impl_glfw
io.BackendRendererName: imgui_impl_opengl3
io.ConfigFlags: 0x00000000
io.ConfigInputTextCursorBlink
io.ConfigWindowsResizeFromEdges
io.ConfigMemoryCompactTimer = 60.0
io.BackendFlags: 0x00000006
 HasMouseCursors
 HasSetMousePos
--------------------------------
io.Fonts: 5 fonts, Flags: 0x00000000, TexSize: 1024,1024
io.DisplaySize: 2560.00,1369.00
io.DisplayFramebufferScale: 1.00,1.00
--------------------------------
style.WindowPadding: 8.00,8.00
style.WindowBorderSize: 1.00
style.FramePadding: 4.00,5.00
style.FrameRounding: 5.00
style.FrameBorderSize: 1.00
style.ItemSpacing: 8.00,6.00
style.ItemInnerSpacing: 4.00,4.00
```

**Version/Branch of Dear ImGui:**

Version: e99c4fc6688e218a0e5da50f56638aebab45da9b `1.89 WIP (18805)`
Branch: master

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_glfw.cpp + imgui_impl_opengl3.cpp
Compiler: MSVC
Operating System: Windows 11

**My Issue/Question:**

Modal popups background is rendered inside modal window in front of modal content.

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/10034350/182128606-a4c42927-b482-457f-b094-27476cf7085a.png)
![image](https://user-images.githubusercontent.com/10034350/182128640-1fdfeeca-a6b3-400f-88b8-8be81262e71d.png)

**Standalone, minimal, complete and verifiable example:** 
```c++
ImGui::ShowDemoWindow(); // "Popups & Modal windows" -> "Modals"
```

**Comment by ocornut** [comment]
A bug was fixed in the OpenGL backend in 1.86 see release note:
https://github.com/ocornut/imgui/releases/tag/v1.86

I am assuming you made an old copy of imgui_impl_opengl3 separate from
imgui core files, and haven’t updated it. You simply need to update it when updating dear imgui. 

Same as #4845 #4887 #4888 #5406 #5438 and more.

**Comment by Grantim** [comment]
Thank you!
